cs4660/graph/graph.py

```diff
@@ -82,27 +105,110 @@ class AdjacencyList(object):
     store nodes and edges
     """
     def __init__(self):
-        # adjacencyList should be a dictonary of node to edges
+        # adjacencyList should be a dictionary of node to edges
         self.adjacency_list = {}
 
     def adjacent(self, node_1, node_2):
+        """
+        * adjacent(node_1, node_2)
+        - returns true if node_1 and node_2 are directly connected or false otherwise
+        """
+        if node_1 not in self.adjacency_list or node_2 not in self.adjacency_list:
+            return False
+
+        for x in self.adjacency_list[node_2]:
+            if node_1 == x.to_node:
+                return True
+        for x in self.adjacency_list[node_1]:
+            if node_2 == x.to_node:
+                return True
```

Comment:
this second for statement is actually wrong because you are assuming the adjacent method is for undirect graph if you also check node2 to node1 in addition to the direction going from node1 to node2.